.github/workflows/release.yml

```
@@ -0,0 +1,73 @@
+name: Release with SLSA provenance
+
+on:
+  push:
+    tags:
+      - 'v*'
+    branches:
+      - 'release/**'
+
+permissions:
+  contents: read
+  packages: write
+  id-token: write # Required for SLSA provenance generation
+
+jobs:
+  build-and-provenance:
+    strategy:
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

from SLSA_Security_Expert
actions/checkout アクションは、SLSAレベル3の要件とセキュリティベストプラクティスに従って、固定のSHAハッシュを使用すべきです。これにより、サプライチェーン攻撃のリスクを軽減できます。

```suggestion
        uses: actions/checkout@8ade135a41bc03ea155e62e844d188df1ea18608 # v4.1.0
```